packages/@aws-cdk/aws-cognito/lib/user-pool.ts

```diff
@@ -1221,6 +1249,27 @@ export class UserPool extends UserPoolBase {
         throw new Error(`Unsupported AccountRecovery type - ${accountRecovery}`);
     }
   }
+
+  private configureUserAttributeChanges(props: UserPoolProps) {
+    let userAttributeUpdateSettings;
+    if (props.keepOriginal) {
+      let attributesRequireVerificationBeforeUpdate: string[] = [];
+
+      if (props.keepOriginal.email) {
+        attributesRequireVerificationBeforeUpdate.push('email');
+      }
+
+      if (props.keepOriginal.phone) {
+        attributesRequireVerificationBeforeUpdate.push('phone');
```

Comment:
As above, there's a constant for that, see L1067 and the string should be `phone_number`
`StandardAttributeNames.phone_number`